src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/VersionCommand.kt

```diff
@@ -0,0 +1,15 @@
+package com.noumenadigital.npl.cli.commands.registry
+
+import java.io.Writer
+
+class VersionCommand : NplCommandExecutor {
+    companion object {
+        const val COMMAND_DESCRIPTION = "Command to return current npl cli version"
+    }
+
+    override val commandName: String = "version"
+
+    override fun execute(output: Writer) {
+        output.write("I'm v1.0")
+    }
```

Comment:
do we have a ticket to properly implement this? if so, let's maybe reference it here with a // TODO ST-XXXX